Apps/APN-UIKit/APN UIKit/APN UIKit/View/SettingsViewController.swift

```diff
@@ -70,10 +87,37 @@ class SettingsViewController: UIViewController {
             handler(settings.authorizationStatus)
         }
     }
+    
+    func compareAndSave() {
+        
+        // Track Url
+        if currentSettings.trackUrl != trackUrlTextField.text {
+            storage.trackUrl = trackUrlTextField.text
+        }
+        
+        // Background Queue Seconds Delay
+        if currentSettings.bgQDelay != bgQTakDelayTextField.text {
+            storage.bgQDelay = bgQTakDelayTextField.text
+        }
+        
+        // Min number of tasks
+        if currentSettings.bgQMinTasks != bgQMinTasksTextField.text {
+            storage.bgNumOfTasks = bgQMinTasksTextField.text
+        }
+        
+        // Min tasks in queue
+        if currentSettings.bgQMinTasks != bgQMinTasksTextField.text {
+            storage.bgNumOfTasks = bgQMinTasksTextField.text
+        }
+        
+        // Track screen enabled
+//        if currentSettings.isTrackScreenEnabled != trackScreenToggle.va
```

Comment:
```suggestion
            storage.trackUrl = trackUrlTextField.text
            storage.bgQDelay = bgQTakDelayTextField.text
            storage.bgNumOfTasks = bgQMinTasksTextField.text
            storage.bgNumOfTasks = bgQMinTasksTextField.text
            ...
```

is there a reason why we are comparing? Couldn't we just save?

Reply:
I don't see any consequences to saving unchanged values. Since the code is more complex by re-saving unchanged values and it makes the code more complex, I suggest we simplify the function by removing the `if` statements.

Reply:
Agreed. It does not make any difference, but simplify the logic. Going ahead & removing the conditions.

Reply:
✅